src/sentry/snuba/metrics/naming_layer/mapping.py

```diff
@@ -63,12 +68,22 @@ def get_public_name_from_mri(internal_name: Union[TransactionMRI, SessionMRI, st
         internal_name = internal_name.value
     assert isinstance(internal_name, str)
 
-    try:
+    if internal_name in MRI_TO_NAME:
         return MRI_TO_NAME[internal_name]
-    except KeyError:
+    elif is_custom_measurement(internal_name):
+        return extract_alias_from_custom_measurement(internal_name)
+    else:
         raise InvalidParams(f"Unable to find a mri reverse mapping for '{internal_name}'.")
 
 
+def is_custom_measurement(internal_name: str) -> bool:
+    return MRI_SCHEMA_REGEX.match(internal_name)
```

Comment:
i don't see any reason to have this function and run `match` two times.

Reply:
I had actually thought about this but I didn't know python supported inline assignments like the one in the `elif`, thanks for letting me know!

Reply:
you are welcome!